poker/src/components/ConnectToLobby/ConnectToLobby.tsx

```diff
@@ -25,79 +26,85 @@ interface Props {
 
 const ConnectToLobby: FC<Props> = ({ setIsVisible, url }) => {
   const history = useHistory();
-  const socket = React.useContext<Socket<DefaultEventsMap, DefaultEventsMap>>(SocketContext);
-
-  const dispatch = useDispatch();
-
   const isAdmin = useSelector(isAdminSlice);
-
-  const room = isAdmin ? `room${socket.id}` : url;
-
-  console.log('SocketId', socket.id);
-
-  const [error, setError] = useState<FormState>({
-    firstName: '',
-    lastName: '',
-    jobPosition: '',
-  });
-
   const [personalData, setPersonalData] = useState<FormState>({
     firstName: '',
     lastName: '',
     jobPosition: '',
   });
   const [observer, setObserver] = useState<boolean>(false);
-  // const [image, setImage] = useState<string>('');
+  const [isError, setIsError] = useState<boolean>(false);
+  const [error, setError] = useState<string>('');
+  const dispatch = useDispatch();
+  const socket = React.useContext<Socket<DefaultEventsMap, DefaultEventsMap>>(SocketContext);
 
-  useEffect(() => {
-    if (personalData.firstName === '') {
-      const { firstName } = personalData;
-      setError((state) => ({ ...state, firstName }));
-    }
-  }, [personalData]);
+  const room = isAdmin ? `room${socket.id}` : url;
+
+  console.log('SocketId', socket.id);
 
   const handleSubmit = (e: SyntheticEvent): void => {
     e.preventDefault();
-    console.log('send form');
-    const { firstName, lastName, jobPosition } = personalData;
-    console.log('room:', room);
-    console.log('url', url);
-
-    let userRole = ROLES.USER;
-    if (observer) userRole = ROLES.OBSERVER;
-    else if (isAdmin) userRole = ROLES.ADMIN;
-
-    const user: Member = {
-      userId: socket.id,
-      firstName,
-      lastName,
-      job: jobPosition,
-      role: userRole,
-    };
-
-    const callback = (response: ResponseFromSocket): void => {
-      console.log(response);
-
-      const { eventName, code, error: responseError, data } = response;
-
-      // eslint-disable-next-line no-console
-      if (responseError) console.log(`${eventName}: ${code}: ${responseError}`);
-      else {
-        const { user: responseUser } = data;
-        dispatch(setCurrentRoom(responseUser.room));
-        const userToRedux = {
-          userId: responseUser.userId,
-          lastName: responseUser.lastName,
-          firstName: responseUser.firstName,
-          job: responseUser.job,
-          role: responseUser.role,
-        };
-        dispatch(loginUser(userToRedux));
-        history.push('/lobby');
-      }
-    };
-
-    socket.emit('login', { user, room }, callback);
+
+    if (personalData.firstName.length > 0) {
+      console.log('send form');
+      const { firstName, lastName, jobPosition } = personalData;
+      console.log('room:', room);
+      console.log('url', url);
```

Comment:
a lot of logs)

Reply:
this is necessary at the development stage, we will remove it for release